--- /home/Workspace/conf-watch/src/repository/home.Workspace.conf-watch.src.tests.1.conf	2011-07-09 19:21:08.486240878 +0700
+++ /home/Workspace/conf-watch/src/tests/1.conf	2011-07-09 19:21:29.246163617 +0700
@@ -55,12 +55,6 @@
   server {
     listen       80;
     client_max_body_size 2000M;
-    error_page   502  /502.html;
-    error_page   405  /405.html;
-    
-    location ~* ^/(502.html|405.html) {
-      root /srv/MFS/MFS/views;
-    }
     
     location /status {
       stub_status on;
